docs/commands.md

```diff
@@ -100,22 +135,22 @@ This can be improved in the future by using binary search to find the start of t
    2) "20"
 
 ```
-## TS.RANGEBYLABELS - ranged query by labels
+## TS.MRANGE - ranged query by filters
 ```sql
-TS.RANGEBYLABELS (labels) fromTimestamp toTimestamp [aggregationType] [bucketSizeSeconds]
+TS.MRABGE fromTimestamp toTimestamp [AGGREGATION aggregationType bucketSizeSeconds] FILTER filter..
 ```
 
-* labels - set of key-pair selector (`k=v`, `k!=v,` `k=` contains a key, `k!=` doesn't contain a key)
 * fromTimestamp - start timestamp for range query
 * toTimestamp - end timestamp for range query
 
 Optional args:
    * aggregationType - one of the following: avg, sum, min, max, count, first, last
    * bucketSizeSeconds - time bucket for aggregation in seconds
+   * filters - set of key-pair fitlers (`k=v`, `k!=v,` `k=` contains a key, `k!=` doesn't contain a key)
 
-### Examples
+### Example
 ```sql
-127.0.0.1:6379> TS.RANGEBYLABELS area_id=32 1548149180 1548149210 avg 5
+127.0.0.1:6379> TS.MRANGE 1548149180 1548149210 AGGREGATION avg 5 FILTER area_id=32 
```

Comment:
Extend the example to have two filter of different types e.g. >=

Reply:
I added a !=